packages-ts/gauntlet-terra-contracts/src/commands/contracts/ocr2/proposeOffchainConfig.ts

```diff
@@ -110,24 +110,26 @@ const makeCommandInput = async (flags: any, args: string[]): Promise<CommandInpu
 }
 
 const beforeExecute: BeforeExecute<CommandInput, ContractInput> = (context) => async () => {
+  // Config in contract
   const event = await getLatestOCRConfigEvent(context.provider, context.contract)
-  const offchainConfig = event?.offchain_config
-    ? await deserializeConfig(Buffer.from(event.offchain_config[0], 'base64'))
+  const offchainConfigInContract = event?.offchain_config
+    ? deserializeConfig(Buffer.from(event.offchain_config[0], 'base64'))
     : ({} as OffchainConfig)
-
-  const contractOffchainConfig = {
-    ...offchainConfig,
-    offchainPublicKeys: offchainConfig.offchainPublicKeys?.map((key) => Buffer.from(key).toString('hex')),
+  const configInContract = longsInObjToNumbers({
+    ...offchainConfigInContract,
+    offchainPublicKeys: offchainConfigInContract.offchainPublicKeys?.map((key) => Buffer.from(key).toString('hex')),
     f: event?.f,
-  }
+  })
```

Comment:
I've seen these LOC a few times in this PR (4x). Please consider extracting this as a helper fn in one place.